docs/design.md

```diff
@@ -8,96 +8,194 @@ Requirements for individual releases can be found on the [Project Page](https://
 Many of the design decisions are inspired by the learnings from the Docker networking design as of Docker v1.6.
 Please refer to this [Docker v1.6 Design](legacy.md) document for more information on networking design as of Docker v1.6.
 
+XXX: Docker v1.6 is very old. Perhaps this is no longer useful and can be removed now?
+
 ## Goal
 
-libnetwork project will follow Docker and Linux philosophy of developing small, highly modular and composable tools that work well independently.
-Libnetwork aims to satisfy that composable need for Networking in Containers.
+The libnetwork project follows the Docker and Linux philosophy of developing small, highly modular and composable tools that work well independently.
+libnetwork aims to satisfy the need for composable networking in containers.
```

Comment:
There were various different capitalisations of libnetwork/Libnetwork/LibNetwork in this document. I changed them all to libnetwork for consistency.